src/data/faq.json

```diff
@@ -467,20 +467,26 @@
+                                    "Unlike the export of individual certificates, the export of all certificates is not limited to those issued in Germany, taking into account all EU digital COVID certificates.",
                                     "<b>Application</b>",
                                     "After successfully exporting the certificate to the PDF document, you can decide whether you want to save it on your smartphone, import it to other apps or print it. To do this, please use the standard functions of the operating system.",
-                                    "<b>Step-by-step instructions:</b>",
-                                    "To create a printout of your EU COVID Digital Certificate, please follow these steps: <ul><li>Tap on the 'Certificates' tab in the Corona-Warn-App</li><li>Select a person, by tapping on a corresponding certificate in the certificate overview</li><li>Scroll down and select a certificate of this person</li><li>In this detail view, tap on the three-dot menu in the upper right corner (Android) or on 'More' at the bottom (iOS)</li><li>Tap on 'Display print version'</li><li>Tap on 'Continue'</li><li>Tap on the icon for 'Print' or for 'Share' in the upper right corner</li><li>In the following view, you can print the created PDF document using the standard functions of the operating system, import it into other apps or even share it</li></ul>"
+                                    "<b>Step-by-step instructions: Exporting a single certificate</b>",
+                                    "To create a printout of your individual EU COVID Digital Certificate, please follow these steps: <ul><li>Tap on the 'Certificates' tab in the Corona-Warn-App</li><li>Select a person, by tapping on a corresponding certificate in the certificate overview</li><li>Scroll down and select a certificate of this person</li><li>In this detail view, tap on the three-dot menu in the upper right corner (Android) or on 'More' at the bottom (iOS)</li><li>Tap on 'Display print version'</li><li>Tap on 'Continue'</li><li>Tap on the icon for 'Print' or for 'Share' in the upper right corner</li><li>In the following view, you can print the created PDF document using the standard functions of the operating system, import it into other apps or even share it</li></ul>",
+                                    "<b>Step-by-step instructions: Exporting all certificates</b>",
+                                    "To create a printout of all your saved EU COVID Digital Certificate at once, please follow these steps: <ul><li>Tap on the 'Certificates' tab in the Corona-Warn-App</li><li>Tap the icon with the upward pointing arrow in the upper right corner</li><li>Tap on 'Continue'</li><li>Tap on the icon for 'Print' or for 'Share' in the upper right corner</li><li>In the following view, you can print the created PDF document using the standard functions of the operating system, import it into other apps or even share it</li></ul>",

```

Comment:
On Android 2.24.x you need to tap on "Next" not "Continue"

Reply:
Fixed in https://github.com/corona-warn-app/cwa-website/pull/2961/commits/7e085cdd9b7b63f28941b34fb0eacc970ebf8d85, thanks for your suggestion